unit_tests/client/test_client.py

```diff
@@ -21,41 +20,48 @@ def child_chain(self):
 
     @pytest.fixture(scope='function')
     def client(self, root_chain, child_chain):
-        return Client(root_chain, child_chain)
+        DUMMY_KEY = '0xa18969817c2cefadf52b93eb20f917dce760ce13b2ac9025e0361ad1e7a1d448'
+        return Client(root_chain, child_chain, DUMMY_KEY)
 
     def test_constructor(self):
         DUMMY_ROOT_CHAIN = 'root chain'
         DUMMY_CHILD_CHAIN = 'child chain'
-        c = Client(DUMMY_ROOT_CHAIN, DUMMY_CHILD_CHAIN)
+        DUMMY_KEY = 'key'
+
+        when('plasma_cash.client.client.utils').normalize_key(DUMMY_KEY).thenReturn(DUMMY_KEY)
+        c = Client(DUMMY_ROOT_CHAIN, DUMMY_CHILD_CHAIN, DUMMY_KEY)
+
         assert c.root_chain == DUMMY_ROOT_CHAIN
         assert c.child_chain == DUMMY_CHILD_CHAIN
+        assert c.key == DUMMY_KEY
+
+    def test_address(self, client):
+        DUMMY_ADDRESS = '0x3B0884f4E50e9BC2CE9b224aB72feA89a81CDF7c'
+
+        assert client.address == DUMMY_ADDRESS
 
     def test_deposit(self, client, root_chain):
         MOCK_TRANSACT = mock()
         DUMMY_AMOUNT = 1
-        DUMMY_DEPOSITOR = 'dummy depositor'
-        DUMMY_CURRENCY = 'dummy currency'
+        DUMMY_CURRENCY = '0x0000000000000000000000000000000000000000'
+        DUMMY_ADDRESS = '0x3B0884f4E50e9BC2CE9b224aB72feA89a81CDF7c'
 
-        when(w3).toChecksumAddress(DUMMY_DEPOSITOR).thenReturn(DUMMY_DEPOSITOR)
         when(root_chain.functions).deposit(DUMMY_CURRENCY, DUMMY_AMOUNT).thenReturn(MOCK_TRANSACT)
 
-        client.deposit(DUMMY_AMOUNT, DUMMY_DEPOSITOR, DUMMY_CURRENCY)
+        client.deposit(DUMMY_AMOUNT, DUMMY_CURRENCY)
 
-        verify(MOCK_TRANSACT).transact(ANY)
+        verify(MOCK_TRANSACT).transact({'from': DUMMY_ADDRESS, 'value': DUMMY_AMOUNT * 10**18})
```

Comment:
I'm actually thinking we can just use `client.address`? and we don't need to declare DUMMY_ADDRESS anymore?